src/chime_dash/app/run.py

```diff
@@ -13,7 +13,6 @@
 SCRIPTS     = Body(LANGUAGE, DEFAULTS).external_scripts
 
 DASH = Dash(
-    __name__,
     external_stylesheets=STYLESHEETS,
```

Comment:
could you please replace __name__ here? i'm not 100% clear why it appears to work just as well either way, but as we start to debug the heroku deploy we'll want it to be in there